[Tournament] Age of Empires 2 (#5)
runebaas/switzerlan-rules: rules/tournaments/aoe2de.md

# Age of Empires 2

## General Provisions

These rules inherit all rules from the main rulebook.

These rules apply to all participants of the "Age of Empires 2" tournament (hereinafter "AoE2" or "Game").

The rules may be available in several languages. If the individual versions differ, the English version is considered the source of the truth.

## Tournament Information

### Tournament Platform

The official tournament platform for this tournament is Battlefy.
Please make sure that you have an account: https://battlefy.com/account/signup

The official tournament page will be shared via the main communication channel established in the main rulebook.

### Tournament Format

The tournament is played in the following stages:

| Stage   | Type               | Format    | Description                                                            |
|---------|--------------------|-----------|------------------------------------------------------------------------|

> | Stage 1 | Swiss Rounds       | Bo1       | This stage will be seeded according to current 1v1 RM ELO rankings.    |

Reviewer: Swiss rounds don't need seeding

| Stage 2 | Single Elimination | Bo1 / Bo3 | The bracket will be seeded according to the result of the first stage. |

### Qualification Swiss System

> 4 rounds are played according to the Swiss-System. The games will be reallocated after each round according to the

Reviewer: I would suggest to rewrite it in the follow way:

> In the Swiss-System tournament, there will be a total of four rounds played. After each round, game pairings will be adjusted based on the current player rankings. These rankings are determined by considering the number of games won and lost by each player. Once the four rounds are completed, the top eight players in the rankings will advance to compete for the tournament victory. In the event of a tie for the eighth place after the four rounds, the ranking will be automatically calculated to determine the final eight participants.

Or potentially drop the entire section and link to the [Wikipedia page](https://en.wikipedia.org/wiki/Swiss-system_tournament).

current ranking. For the ranking the won and lost games count. The best eight players will then play for the tournament
victory. If there is a tie for 8th place after the 4 rounds, the ranking will be calculated automatically.

## Schedule

The first appointment for the team representative will be the briefing on Saturday at 15:30.
The tournament itself will start one hour later at 16:30.
All players must be at the location at the start of the tournament.

> No accommodations will be made for other tournaments that may happen during the time of this tournament.

Reviewer: This line is only valid for T1 tournaments, Other tiers should accommodate those.

## Settings

### Game Version

The game is played with the **latest version** of Age of Empires 2: Definitive Edition.

### Server

The game is played on the official "westeurope" server.

### Lobby Settings

The following settings must be set and checked before each game.

![Lobby Settings for SwitzerLAN 23](https://cdn.discordapp.com/attachments/592823339643437056/1158747859139710987/lobby_settings.png "Lobby Settings")

### Game Settings

The following settings must be set and checked before each game.

![Game Settings for SwitzerLAN 23](https://cdn.discordapp.com/attachments/592823339643437056/1158747858854482022/game_settings.png "Game Settings")

### Mods

> All mods are allowed.

Reviewer: Wouldn't this lead to cheating?

Author: So far in AoE2 there has not been a mod that has unfairly affected the game. I therefore assume that there is currently no mod that could be considered cheating. However, I have clarified the text a bit to be prepared for all cases.

## Game Account

> Each player plays with his own game account.

Reviewer: ```suggestion
Each player plays with their own game account.
```

## Map Pool

* Arabia
* Arena
* African Clearing
* Gold Rush
* Four Lakes
* Acropolis

### Map Selection Process

#### Swiss Rounds

> In the first round, the lower ranked (by 1v1 RM ELO) player has the choice of which map is played.

Reviewer: Swiss rounds don't need seeding, so this should probably rewritten as

```suggestion
In the first round, the last named player can pick the map. 
```

Or alternatively, to make it fair, map banning or a roll of the dice 🎲

For the next rounds the player with the lower current ranking in the Swiss Stage standings chooses the map.

#### Single Elimination

In a best-of-1 match, the lower seeded player chooses the map.
In a best-of-3 match, the first game is always played on Arabia.
Afterward the loser of the previously played game chooses the map.

## Civilization Draft

Players may pick any civilization, but may not repeat any civilization in a match. This means that if, for example,
a match lasts for three games, players will have played three different civilizations each (also known as "hidden pick,
no repeat").

> ## Conditions of Participation

Reviewer: This section is already covered in the main rulebook, no need to repeat...

Each tournament participant is responsible...

...that he/she attends the briefing on time.  
...that he/she is ready for the tournament games.  
...that he/she is registered for the tournament on the [SwitzerLAN website](https://switzerlan.ch/turnier/?id=287) as well as on Battlefy.  
...that he/she joins the [SwitzerLAN Discord](https://discord.com/invite/AwJb9mF) server and responding to messages on there.

## Playing Situations

> ### Pausing the Game

Reviewer: This is already covered in the main rulebook, no need to repeat...

When pausing the game, the reason and duration of the pause should always be briefly discussed among each other.

> ### Restore the Game

Reviewer: ```suggestion
### Disconnect or game crash
```

Then you can drop everything before the first comma in the paragraph bellow

In case of a disconnect or game crash, the player remaining in the game should save the game. The game should then be
restored to the last available point in time.

### End of Game

The winner is the player who has the most points after 700 years (60 in-game minutes). Of course, a player also wins if
the opponent surrenders prior to the time limit.

> ### Winning the Game / Results

Reviewer: This is already covered in the main rulebook, no need to repeat...
